client-e2e/test/specs/pages.ts

```diff
@@ -8,7 +8,7 @@ describe("Check internal Pages", () => {
         "Send files in real-time"
       );
 
-      await $('a[href="/about"').click();
+      await $('a[to="/about"').click();
```

Comment:
Seems there was a mistake in the `Link` component. Links should render `<a href="">` and not `<a to="">`. I fixed it here: https://github.com/LeastAuthority/winden/pull/133

Once it gets merged, we can revert this change to use `a[href=""]` again